docs/pages/services/cassandra/limitations.md

```diff
@@ -0,0 +1,292 @@
+---
+post_title: Limitations
+menu_order: 80
+feature_maturity: preview
+enterprise: 'no'
+---
+
+* Seed Node Replace
+
+While the Cassandra service supports a node replace operation via the command
+`dcos cassandra pods replace ≤node_id≥`, the node replace operation does not
+currently inform the underlying Cassandra cluster of the change, so effectively
```

Comment:
Are you suggesting that the statement would be strictly true, so accepted, if I change it to qualify as "... the node replace operation _for a seed node_ does not ...."?